activesupport/test/cache/stores/null_store_test.rb

```diff
@@ -62,6 +62,19 @@ def test_local_store_strategy
     assert_nil @cache.read("name")
   end
 
+  def test_namespaced_local_store_strategy
```

Comment:
These tests should go in `activesupport/test/cache/behaviors/local_cache_behavior.rb`.  The reason `test_local_store_strategy` and `test_local_store_repeated_reads` are defined here is because `NullStoreTest` does not include `LocalCacheBehavior` (but the Redis and Memcache tests do).

Reply:
are you saying you DO want `LocalCacheBehavior` to `include LocalCacheBehavior` ?

Reply:
> You mean "NullStoreTest to include LocalCacheBehavior"?

yeah... I corrected my comment when I saw it 🤦🏻

gotcha... that makes sense and may in fact find other issues (still testing)

Reply:
> and may in fact find other issues

yeah, including the repeated reads test in the local cache behaviour exposes further issues related to how it handles nil or not found values... more digging to do